tool/tsh/common/tsh.go

```diff
@@ -1032,6 +1040,7 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	appLogin := apps.Command("login", "Retrieve short-lived certificate for an app.")
 	appLogin.Arg("app", "App name to retrieve credentials for. Can be obtained from `tsh apps ls` output.").Required().StringVar(&cf.AppName)
 	appLogin.Flag("aws-role", "(For AWS CLI access only) Amazon IAM role ARN or role name.").StringVar(&cf.AWSRole)
+	appLogin.Flag("env", "(For AWS CLI access only) Obtain credentials as plain text loads them into environments variables. Required when using per-session MFA.").BoolVar(&cf.AppLoginAWSEnvOutput)
```

Comment:
nit: sentence doesn't quite flow, maybe:
```suggestion
	appLogin.Flag("env", "(For AWS CLI access only) Obtain credentials as plain text in order to load into environments variables. Required when using per-session MFA.").BoolVar(&cf.AppLoginAWSEnvOutput)
```